src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.jsx

```diff
@@ -74,6 +73,12 @@ const organizationNavigation = [
         show: ({access}) => access.has('org:integrations'),
         description: t('Manage integrations for an organization'),
       },
+      {
+        path: `${pathPrefix}/developer-settings/`,
+        title: t('Developer Settings'),
+        show: ({access, features}) => features.has('internal-catchall'),
```

Comment:
Are we not planning on adding a specific feature flag for this?

Reply:
I don't know if that's ironed out yet - might just be `org:integrations` for access